compiler/test/dotty/tools/vulpix/SummaryReport.scala

```diff
@@ -0,0 +1,94 @@
+package dotty
+package tools
+package vulpix
+
+import scala.collection.mutable
+import dotc.reporting.TestReporter
+
+trait SummaryReporting {
```

Comment:
Could you add some documentation in this file, what's a SummaryReporting? What's a SummaryReport? Also, the fact that these are two different classes with very similar name is confusing, maybe use SummaryReport and DefaultSummaryReport; I'm not sure.